### ISSUE TITLE
Reconsider " If this is a bug, please file an issue at https://github.com/GoogleContainerTools/jib/issues/new"

### ISSUE THREAD

**Issue by elharo** [GoogleContainerTools/jib#3283]
In about six files. Most cases seem unlikely to be bugs.

**Comment by mpeddada1** [comment]
Thanks, that is a good point. The word "bug" can be misleading because it may not always be the case that something is broken. Perhaps we can rephrase it to "If you need more assistance with this, please file an issue ..."

**Comment by elharo** [comment]
I'm not sure I even want to go that far. Given current support for jib, I'd prefer not to create extra issues in the repo and an expectation of unpaid support. I'm happy to receive genuine bugs here, but I'd prefer to leave it at that.

**Comment by elharo** [comment]
Plus stack overflow is a better channel for general questions than Github.

**Comment by chanseokoh** [comment]
Yeah, time to rephrase/remove them. They were put in the old days where we were not GA and a lot of things were unsupported and unknown.

**Comment by JacekDuszenko** [comment]
Hey, I'd like to take a stab at this :)

**Comment by elharo** [comment]
Thanks, You can send the review to me if you like. CLA will be required. Google practices TDD so be sure to write or modify tests first before changing the model code.

**Comment by JacekDuszenko** [comment]
@elharo  I've just submitted a PR: #3294 
Feel free to take a look in your free time
Thanks!
